src/main/java/com/hadoop/compression/lzo/DistributedLzoIndexer.java

```diff
@@ -25,57 +27,108 @@
 
 public class DistributedLzoIndexer extends Configured implements Tool {
   private static final Log LOG = LogFactory.getLog(DistributedLzoIndexer.class);
-  private final String LZO_EXTENSION = new LzopCodec().getDefaultExtension();
 
+  private static final String LZO_EXTENSION = new LzopCodec().getDefaultExtension();
+
+  private static final String LZO_SKIP_INDEXING_SMALL_FILES_KEY = "lzo_skip_indexing_small_files";
```

Comment:
Nit: I think it's a common practice for hadoop and related code bases to use dots (".") as separators for config keys; e.g. "lzo_skip_indexing_small_files" -> "lzo.skip-indexing-small-files". By the same token, how about "lzo.skip-indexing-small-files.size" instead of "lzo_small_file_size", and "lzo.recursive-indexing.enabled" instead of "lzo_recursive_indexing"?

Another nit: let's pair each key definition and its default.

Another nit: have an empty line between the static members and the instance members.